terraform/cp4ba/variables.tf

```diff
@@ -65,41 +68,45 @@ variable "owner" {
   description = "Ignored if `cluster_id` is specified. Use your user name or team name. The owner is used to label the cluster and other resources with the tag 'owner:{owner}'"
 }
 
-variable "on_vpc" {
-  default     = false
-  description = "Select 'true' to install on a VPC cluster and it's using VPC Gen2. Note: CP4BA does not currently support VPC cluster."
-}
+//variable "on_vpc" {
+//  default     = false
+//  description = "Select 'true' to install on a VPC cluster and it's using VPC Gen2. Note: CP4BA does not currently support VPC cluster."
+//}
+
+//variable "entitlement" {
```

Comment:
This shouldn't be commented out.  And should be defaulted to "cloud_pak".

Reply:
`entitlement` uncommented`